src/Microsoft.AspNetCore.SignalR.Core/HubConnectionList.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
-    public class HubConnectionList : IReadOnlyCollection<HubConnectionContext>
+    public class HubConnectionList
```

Comment:
Having a public class that is called a list but doesn't implement a collection interface feels wrong.

Reply:
Meh.